packages/agoric-cli/src/lib/chain.js

```diff
@@ -132,6 +133,7 @@ harden(execSwingsetTransaction);
 /**
  *
  * @param {MinimalNetworkConfig} net
+ * @returns {ParamsSDKType}
```

Comment:
I'm not sure JSON output from `agd` always directly corresponds with protobuf stuff. I wouldn't be surprised if it does in this case, but did you happen to verify experimentally? Or maybe there's a caller that clearly relies on it?

Reply:
Yeah there's a caller. 
<img width="456" alt="Screenshot 2025-01-24 at 2 41 02 PM" src="https://github.com/user-attachments/assets/86ccbb14-d0df-478b-ba51-b96607106afe" />